parsl/dataflow/strategy.py

```diff
@@ -189,9 +194,11 @@ def _strategy_simple(self, tasks, *args, kind=None, **kwargs):
             nodes_per_block = executor.provider.nodes_per_block
             parallelism = executor.provider.parallelism
 
-            running = sum([1 for x in status if x == 'RUNNING'])
-            submitting = sum([1 for x in status if x == 'SUBMITTING'])
-            pending = sum([1 for x in status if x == 'PENDING'])
+            running = sum([1 for x in status.values() if x.state == JobState.RUNNING])
+            # I didn't see any provider setting the status to 'SUBMITTING'
```

Comment:
Me neither. I think the only references of this value are the `active_blocks` calculation and log messages a few lines later.

It would be a reasonable separate PR to remove submitting from here.

Reply:
If it's reasonable here, there is no need for additional bureaucracy.

Reply:
state:submitting made sense if we were to use `execute_no_wait` calls on batch systems that take minutes to respond. It's obsolete now.

Reply:
Marking this as resolved.